///////////////////////////////////////////////////////////////////////////////////////////////////
///
/// @file      ProductSTS.proto
///
/// @brief     Data types used by the Product STS controller
///
/// @attention Copyright 2018 Bose Corporation
///
///////////////////////////////////////////////////////////////////////////////////////////////////

import "ProtoToMarkup/MarkupOptions.proto";

package ProductSTS;

///////////////////////////////////////////////////////////////////////////////////////////////////
///
/// This is the identifier used to uniquely designate a product source slot, as well as a source
/// and source account name, based on the associated enumeration string, for Castle content items.
///
///////////////////////////////////////////////////////////////////////////////////////////////////
enum ProductSourceSlot
{
    PRODUCT = 0;
    SETUP   = 1;
    AUX     = 2;
}

////////////////////////////////////////////////////////////////////////////////////////////////////
///                                        End of File                                           ///
////////////////////////////////////////////////////////////////////////////////////////////////////
